service-base/src/main/java/org/eclipse/hono/service/credentials/CredentialsHttpEndpoint.java

```diff
@@ -135,6 +130,7 @@ private void updateCredentials(final RoutingContext ctx) {
         final String deviceId = payload.getString(CredentialsConstants.FIELD_PAYLOAD_DEVICE_ID);
         final String authId = payload.getString(CredentialsConstants.FIELD_AUTH_ID);
         final String type = payload.getString(CredentialsConstants.FIELD_TYPE);
+        final String resourceVersion = ctx.get(KEY_ETAGS_VERSION);
```

Comment:
if we use the term *resource version* in our code, then IMHO we should use this name for the key as well, i.e. `KEY_RESOURCE_VERSION`. The fact that it is related to the ETAG value returned by the service and that its value has been retrieved from the If-Match header field should be an implementation detail of the *extractIfMatchVersionParam* method ...